### PR TITLE
Created home screen, register screen and added user header in menu drawer.

### PR DESCRIPTION
**Pull request by oladyr**

### CODE REVIEW

File: lib/screens/BookDetailsScreen.dart [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import 'dart:html';
+//import 'dart:html';
```
Comment (kaes1) [inline lib/screens/BookDetailsScreen.dart]: Remove unncessary comment.

File: lib/MenuDrawer.dart [hunk 2]
```diff
@@ -59,6 +68,32 @@ class MenuDrawer extends StatelessWidget {
...
 
+  Widget _createDrawerHeader(IconData icon, String text){
+    return UserAccountsDrawerHeader(
+      currentAccountPicture: new CircleAvatar(
+          backgroundColor: Colors.white,
+          //backgroundColor: Colors.deepPurple,
+          child: new Icon(
+            icon,
+            color: Colors.deepPurple[400],
+            //color:Colors.white,
```
Comment (kaes1) [inline lib/MenuDrawer.dart:79]: Remove comments.

File: lib/ApiService.dart [hunk 3]
```diff
@@ -5,7 +5,8 @@ import 'package:http/http.dart';
 import 'package:http/http.dart' as http;
 
 class ApiService {
-  final Uri serverUri = Uri.parse('http://192.168.0.120:8080');
+  //final Uri serverUri = Uri.parse('http://192.168.0.103:8080');
+  final Uri serverUri = Uri.parse('http://192.168.43.71:8080');
```
Comment (kaes1) [inline lib/ApiService.dart:12]: Add comments describing IPs.

**Review by kaes1 — CHANGES_REQUESTED** [review]

File: lib/MenuDrawer.dart (hunk 2 above)
Comment (oladyr) [reply lib/MenuDrawer.dart:79]: Comments contain alternative colors for user header.

**Review by kaes1 — APPROVED** [review]

**Pull request merged**